extensions/arc/src/ui/dashboards/postgres/postgresOverviewPage.ts

```diff
@@ -281,6 +342,59 @@ export class PostgresOverviewPage extends DashboardPage {
 		];
 	}
 
+	private getPodStatus(): PodStatusModel[] {
+		let podModels: PodStatusModel[] = [];
+		const podStatus = this._postgresModel.config?.status.podsStatus;
+
+		podStatus?.forEach(p => {
+			// If a condition of the pod has a status of False, pod is not Ready
+			let status = loc.ready;
+			p.conditions.forEach(c => {
```

Comment:
Don't you want to filter this to only conditions with the type `Ready`?

Reply:
Went with the idea that if all conditions are not met, pod is not ready.
*There should be more done to calculate the status of the pods by aggregating the 4 conditions in the future.